test/unit/spec/MfaVerify_spec.js

```diff
@@ -2178,7 +2178,7 @@ function (Okta,
               var form = test.form[1];
               form.inlineTOTPAdd().click();
               expect(form.inlineTOTPAdd().length).toBe(0);
-              Expect.isTextField(form.answerField());
+              Expect.isTelField(form.answerField());
```

Comment:
can we use the same function as in the other tests?  ```expectHasAnswerField```? That way we don't need to implement a new method in ```Expect``` that does the same thing.

Reply:
Done